docarray/array/mixins/proto.py

```diff
@@ -24,12 +36,35 @@ def to_protobuf(self) -> 'DocumentArrayProto':
         :return: the protobuf message
         """
 
-        from docarray.proto import DocumentArrayProto
+        from docarray.proto import (
+            DocumentArrayListProto,
+            DocumentArrayProto,
+            DocumentArrayStackedProto,
+            UnionArrayProto,
+        )
+
+        if self.is_stacked() and self._columns is not None:
```

Comment:
I think this means that if it is stacked, but columns are non, then serializing and de-serializing will yield a da that is now not stacked, right? is that what we want?

Reply:
actually `self._columns is not None` is only here because of mypy. self._columns cannot be None if we are in stacked mode but mypy cannot know.